### DISCUSSION TITLE
OpenCV(4.5.2) C:\Users\runneradmin\AppData\Local\Temp\pip-req-build-_8k9tw8n\opencv\modules\imgproc\src\color.cpp:182: error: (-215:Assertion failed) !_src.empty() in function 'cv::cvtColor'

### DISCUSSION THREAD

**Discussion by HrutujaDiwane** [datasciencecompleteguide/Machine-Learning#13] [Q&A]
![image](https://user-images.githubusercontent.com/82497663/124438421-cc287080-dd95-11eb-98dd-b76ad4f4f2e4.png)

**Comment by datasciencecompleteguide** [comment]
You r trying to convert the color of the image that is not yet loaded into the "img" variable. Thus, try to read the image properly. 

If possible then share the dataset also so that i can debug the code that u have shared.

Without the data set it won't be possible for me to debug it completely.

**Reply by HrutujaDiwane** [reply]
https://www.kaggle.com/c/aptos2019-blindness-detection/data                                                                                                                  this dataset we use.

**Reply by datasciencecompleteguide** [reply]
> 
> 
> https://www.kaggle.com/c/aptos2019-blindness-detection/data this data-set we use.

This is the real-time competition. I cannot help u with the link because for that i have to accept the terms and conditions of the competition. So, please share the files directly (if you can) otherwise (if you want privacy and cannot share the code and dataset) then, the hint is that there is the problem in reading the image. So, check the dimensions of the dataset and check the dimensions of the array that "imread" function accepts.
